src/xcode/ENA/ENA/Source/Services/TicketValidation/Models/TicketValidationAccessToken.swift

```diff
@@ -18,10 +19,10 @@ import Foundation
 // | `t` | int | The type of the validation (0 = Structure, 1 = Cryptographic, 2 = Full) |
 // | `vc` | object | A data structure representing the validation conditions (see below) |
 
-struct TicketValidationAccessToken: Codable {
+struct TicketValidationAccessToken: Equatable, Codable, Claims {
 	let iss: String
-	let iat: Int
-	let exp: Int
+	let iat: Date?
```

Comment:
Will this be decoded correctly from Int to Date?

Reply:
We checked the decoder, dateDecodingStrategy is set correct.